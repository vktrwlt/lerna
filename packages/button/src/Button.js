import styled from "styled-components";
import { colors } from "@lerna-ds/design-tokens";

export const Button = styled.button`
  background-color: ${colors.green};
  color: #000;

  border: none;
  border-radius: 8px;

  padding: 16px 32px;
  font-size: 16px;
  font-family: Verdana, Geneva, Tahoma, sans-serif;

  cursor: pointer;

  transition: 0.2s ease-in-out;

  &:hover {
    background: #0fff00;
  }
`;
