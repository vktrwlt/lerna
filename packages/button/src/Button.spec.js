import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import { Button } from "./Button";

describe("Button", () => {
  test("render component", () => {
    const tree = renderer.create(<Button> Test </Button>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
