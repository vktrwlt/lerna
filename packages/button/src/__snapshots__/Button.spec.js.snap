// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Renderizou corretamente 1`] = `
<button
  className=""
>
   Teste 
</button>
`;
